pkg/util/util.go

```diff
@@ -39,3 +40,28 @@ func ValidateDriverName(driverName string) error {
 	}
 	return err
 }
+
+func CleanTargetPath(targetPath string) {
+	// remove the targetPath parent directory
+	// resolve kubelet issue https://github.com/kubernetes/kubernetes/issues/60987
+	_ = os.Remove(getParentDirectory(targetPath))
```

Comment:
Returning the error is important, it should get logged.

Reply:
Thanks @nixpanic for review. I fix the comments that you reviewed. but I commit wrong file, so I have to close this PR for a new PR